tests/main/cgroup-devices/task.yaml

```diff
@@ -1,6 +1,7 @@
 summary: measuring basic properties of device cgroup
 # fedora-31: uses cgroupv2, which we don't support
-systems: [ -fedora-31-*]
+# fedora-32: uses cgroupv2, which we don't support
```

Comment:
Specifically it's not that we don't support it, it's just not supported in cgroupv2. That's not a biggie though, I will get to this eventually.